gradle/scripts/lib/java-rpc-proto.gradle

```diff
@@ -36,6 +36,11 @@ configure(projectsWithFlags('java')) {
                             artifact = "com.salesforce.servicelibs:rxgrpc:${managedVersions['com.salesforce.servicelibs:reactor-grpc']}"
                         }
                     }
+                    if (project.ext.hasFlag('kotlin-grpc')) {
+                        kotlinGrpc {
+                            artifact = "io.grpc:protoc-gen-grpc-kotlin:${managedVersions['io.grpc:protoc-gen-grpc-kotlin']}"
+                        }
+                    }
```

Comment:
Could you send a pull request for this to line/gradle-scripts?

Reply:
Since the `protoc-gen-grpc-kotlin` doesn't run well on Windows, suggest instead of tying the build to what the compiler supports, in this case just checking in the generated code.

Reply:
Then we could still check this code in to line/gradle-scripts and remove the `'kotlin-grpc'` flag from `settings.gradle`.

Reply:
> Could you send a pull request for this to line/gradle-scripts?

Sure. let me make a PR.

Reply:
Time to merge master